src/tactic/slim_check.lean

```diff
@@ -184,8 +185,22 @@ meta def slim_check (cfg : slim_check_cfg := {}) : tactic unit := do
 { tgt ← retrieve $ tactic.revert_all >> target,
   let tgt' := tactic.add_decorations tgt,
   let cfg := { cfg with enable_tracing := cfg.enable_tracing || is_trace_enabled_for `slim_check.discared },
-  e ← mk_mapp ``testable.check [tgt, `(cfg), tgt', none],
-  `(@testable.check _ _ _ %%inst) ← pure e,
+  inst ← mk_app ``testable [tgt'] >>= mk_instance <|>
+    fail!"Failed to create a `testable` instance for `{tgt}`.
+What to do:
+1. make sure that the types you are using have `sampleable` instances (you can use `#sample my_type` if you are unsure);
+2. make sure that the relations and predicates that your proposition use are decidable;
+3. make sure that instances of `testable` exist that, when combined, apply to your decorated proposition:
+```
+{tgt'}
+```
+
+Use `set_option trace.class_instances true` to understand what instances are missing.
+
+Try this:
```

Comment:
Does this usage of ``Try this:` actually work? @bryangingechen @EdAyers

Reply:
No, but I don't think it's meant to since it wouldn't make any sense to replace the tactic call `slim_check` with  
```lean
set_option trace.class_instances true
#check (by apply_instance : testable ({tgt'}))
```
and it doesn't cause any problems either, since [the regex in vscode-lean](https://github.com/leanprover/vscode-lean/blob/master/src/trythis.ts#L3) looks for stuff between "Try this:" and the next unindented line (I also tested one of the examples from #4022 with this branch and this error message didn't trigger a suggestion).